api/swagger.yaml

```diff
@@ -4443,6 +4443,8 @@ paths:
       responses:
         200:
           description: "no error"
+        404:
+          description: "repository not found"
```

Comment:
Should we include the "or no read access" part of this error? It may be important information to mention here, because the repository _may_ exist, just return a 404 if there's not access to read it

/cc @dmcgowan